java/openinference-semantic-conventions/src/main/java/io/openinference/semconv/trace/SpanAttributes.java

```diff
@@ -0,0 +1,116 @@
+    public static final AttributeKey<String> OUTPUT_MIME_TYPE = AttributeKey.stringKey("output.mime_type");
+    
+    // Input attributes
+    public static final AttributeKey<String> INPUT_VALUE = AttributeKey.stringKey("input.value");
+    public static final AttributeKey<String> INPUT_MIME_TYPE = AttributeKey.stringKey("input.mime_type");
+    
+    // Embedding attributes
+    public static final AttributeKey<String> EMBEDDING_EMBEDDINGS = AttributeKey.stringKey("embedding.embeddings");
+    public static final AttributeKey<String> EMBEDDING_MODEL_NAME = AttributeKey.stringKey("embedding.model_name");
+    
+    // LLM attributes
+    public static final AttributeKey<String> LLM_FUNCTION_CALL = AttributeKey.stringKey("llm.function_call");
+    public static final AttributeKey<String> LLM_INVOCATION_PARAMETERS = AttributeKey.stringKey("llm.invocation_parameters");
+    public static final AttributeKey<String> LLM_INPUT_MESSAGES = AttributeKey.stringKey("llm.input_messages");
+    public static final AttributeKey<String> LLM_OUTPUT_MESSAGES = AttributeKey.stringKey("llm.output_messages");
+    public static final AttributeKey<String> LLM_MODEL_NAME = AttributeKey.stringKey("llm.model_name");
+    public static final AttributeKey<String> LLM_PROVIDER = AttributeKey.stringKey("llm.provider");
+    public static final AttributeKey<String> LLM_SYSTEM = AttributeKey.stringKey("llm.system");
+    public static final AttributeKey<String> LLM_PROMPTS = AttributeKey.stringKey("llm.prompts");
+    public static final AttributeKey<String> LLM_PROMPT_TEMPLATE = AttributeKey.stringKey("llm.prompt_template.template");
+    public static final AttributeKey<String> LLM_PROMPT_TEMPLATE_VARIABLES = AttributeKey.stringKey("llm.prompt_template.variables");
+    public static final AttributeKey<String> LLM_PROMPT_TEMPLATE_VERSION = AttributeKey.stringKey("llm.prompt_template.version");
+    
+    // LLM request parameters
+    public static final AttributeKey<Double> LLM_REQUEST_TEMPERATURE = AttributeKey.doubleKey("llm.request.temperature");
+    public static final AttributeKey<Long> LLM_REQUEST_MAX_TOKENS = AttributeKey.longKey("llm.request.max_tokens");
+    public static final AttributeKey<Double> LLM_REQUEST_TOP_P = AttributeKey.doubleKey("llm.request.top_p");
+    public static final AttributeKey<Double> LLM_REQUEST_TOP_K = AttributeKey.doubleKey("llm.request.top_k");
+    public static final AttributeKey<Double> LLM_REQUEST_FREQUENCY_PENALTY = AttributeKey.doubleKey("llm.request.frequency_penalty");
+    public static final AttributeKey<Double> LLM_REQUEST_PRESENCE_PENALTY = AttributeKey.doubleKey("llm.request.presence_penalty");
+    public static final AttributeKey<String> LLM_REQUEST_STOP_SEQUENCES = AttributeKey.stringKey("llm.request.stop_sequences");
+    public static final AttributeKey<Long> LLM_REQUEST_SEED = AttributeKey.longKey("llm.request.seed");
```

Comment:
I don't think these are correct - it's just a JSON dump of all the params. Can you double check?